package.json

```diff
@@ -16,7 +16,7 @@
         "assert-plus": "^1.0.0",
         "backoff": "2.3.0",
         "bignum": "0.13.0",
-        "buckets-mdapi": "git+https://github.com/joyent/node-buckets-mdapi.git#v0.5.0",
+        "buckets-mdapi": "git+https://github.com/chudley/node-buckets-mdapi.git#dev-MANTA-4341",
```

Comment:
I approved the `node-buckets-mdapi` PR so just a note to update this once that's merged.